cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebViewHelper.java

```diff
@@ -95,6 +99,60 @@ public void run() {
         });
     }
 
+    @TargetApi(11)
```

Comment:
I checked the [doc](https://developer.android.com/reference/android/annotation/TargetApi.html), but i don't quite understand what's the effect of the annotation. Any explanation is appreciated.